c7n/resources/ses.py

```diff
@@ -87,6 +87,34 @@ def process(self, resources):
             )
 
 
+@SESConfigurationSet.action_registry.register('delete')
+class DeleteSESConfigurationSet(Action):
+    """Delete an SES Configuration Set resource.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: ses-delete-configuration-set
+                resource: aws.ses-configuration-set
+                actions:
+                    - delete
+
+    """
+    schema = type_schema('delete')
+    permissions = ("ses:DeleteConfigurationSet",)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('ses')
+        for resource in resources:
+            self.manager.retry(
+                client.delete_configuration_set,
+                ConfigurationSetName=resource['Name'],
+                ignore_err_codes=("CannotDeleteException",)
```

Comment:
Based on boto3 doc: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses/client/delete_configuration_set.html, there is not "CannotDeleteException", only "ConfigurationSetDoesNotExistException".

Reply:
@jerryhxu changed it to correct exception. thank you.